src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/ComponentFactoryBase.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tizen.Applications.CoreBackend;
+using static Interop.CBApplication;
+using static Tizen.Applications.CoreBackend.DefaultCoreBackend;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    internal abstract class ComponentFactoryBase
+    {
+        internal readonly Type _classType;
```

Comment:
Do not directly open a field. 

If you need to open this information you should provide as property

```suggestion
        public Type ComponentType { get; }
```

Reply:
Done